sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -353,6 +355,7 @@ public partial class BlobContainerProperties
         public System.DateTimeOffset? DeletedOn { get { throw null; } }
         public Azure.ETag ETag { get { throw null; } }
         public bool? HasImmutabilityPolicy { get { throw null; } }
+        public bool HasImmutableStorageWithVersioning { get { throw null; } }
```

Comment:
how is this different from `HasImmutabilityPolicy` line above?

Reply:
GET https://seanoauthstage.blob.core.windows.net/test-container-9d4eb5b6-8be5-4d1e-ba83-ef2c60f8199e?restype=container HTTP/1.1
x-ms-version: 2020-08-04
Accept: application/xml
x-ms-client-request-id: 2fc22202-060c-4a0b-be5b-9d9e5410f387
x-ms-return-client-request-id: true
User-Agent: azsdk-net-Storage.Blobs/12.9.0-alpha.20210604.1 (.NET Framework 4.8.4300.0; Microsoft Windows 10.0.19043 )
x-ms-date: Fri, 04 Jun 2021 18:06:23 GMT
Authorization: [redacted]
traceparent: 00-5d424cc99718c34f9a124b2cf9e42577-3f00ee44eb443749-00
Host: seanoauthstage.blob.core.windows.net

HTTP/1.1 200 OK
Transfer-Encoding: chunked
Last-Modified: Fri, 04 Jun 2021 18:06:22 GMT
ETag: "0x8D92783760EC0E8"
Server: Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0
x-ms-request-id: 4e6de559-c01e-0030-466c-59b723000000
x-ms-client-request-id: 2fc22202-060c-4a0b-be5b-9d9e5410f387
x-ms-version: 2020-08-04
x-ms-lease-status: unlocked
x-ms-lease-state: available
**x-ms-has-immutability-policy: false**
x-ms-has-legal-hold: false
**x-ms-immutable-storage-with-versioning-enabled: true**
x-ms-blob-public-access: container
x-ms-default-encryption-scope: $account-encryption-key
x-ms-deny-encryption-scope-override: false
Date: Fri, 04 Jun 2021 18:06:22 GMT

0